pkg/skr/gcpnfsvolumerestore/checkStates.go

```diff
@@ -0,0 +1,39 @@
+package gcpnfsvolumerestore
```

Comment:
Minor: checkStates is very vague and doesn't really explain what this action do. This is just to set the Processing state if it was in error (which is not a final) state.

Reply:
Changed the name to `setProcessing`.